test/transaction.ts

```diff
@@ -558,7 +575,7 @@ describe('Transaction', () => {
         const {gaxOpts, reqOpts} = REQUEST_STREAM.lastCall.args[0];
 
         assert.strictEqual(reqOpts.gaxOptions, undefined);
-        assert.strictEqual(gaxOpts, fakeQuery.gaxOptions);
+        assert.deepStrictEqual(gaxOpts, fakeQuery.gaxOptions);
```

Comment:
```suggestion
        assert.strictEqual(gaxOpts, fakeQuery.gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `fakeQuery.gaxOptions`.